modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/AdminTooltip.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayAutocomplete from '@clayui/autocomplete';
+import ClayButton from '@clayui/button';
+import ClayCard from '@clayui/card';
+import {useResource} from '@clayui/data-provider';
+import ClayDropDown from '@clayui/drop-down';
+import ClayForm, {ClayInput, ClayRadio, ClayRadioGroup} from '@clayui/form';
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+
+const AdminTooltip = ({setRemovePinHandler, setShowTooltip, showTooltip}) => {
+	const [position, setPosition] = useState(showTooltip.details.label);
+	const [linkedValue, setLinkedValue] = useState(
+		showTooltip.details.linked_to_sku
+	);
+	const [sku, setSku] = useState(showTooltip.details.sku);
+	const [quantity, setQuantity] = useState(showTooltip.details.quantity);
+
+	const [networkStatus, setNetworkStatus] = useState(4);
+
+	const initialLoading = networkStatus === 1;
+	const loadingAd = networkStatus < 4;
+	const error = networkStatus === 5;
+
+	const {resource} = useResource({
+		fetchPolicy: 'cache-first',
+		link: 'https://rickandmortyapi.com/api/character/',
```

Comment:
I like this API, but is it supposed to be merged into master?

Reply:
We usually use "example.com" for URLs

Reply:
Responding in order: yes it's supposed to be merged in master sooner (or later). This module will not be startable neighter visualizable in Liferay before step 3of5 when we should merge together this workin component with the 3+ modules for the backend logic and apis (blocked at this time so still in progress). This api call it's not usable since we test and integrate the api part. In the end i just copied as is this component from clay only for a complete-visual check with the designs. should i use https://dummy.restapiexample.com/ maybe? ty

Reply:
Thanks for the reply @garofalog, if this is temporary I think you can use what you want, forget my previous comment